etna/media/blocks.py

```diff
@@ -34,6 +34,9 @@ def get_api_representation(self, value, context=None):
             "date": value.date,
             "description": expand_db_html(value.description),
             "transcript": expand_db_html(value.transcript),
+            "chapters": value.chapters.stream_block.get_api_representation(
+                value.chapters, context
+            ),
```

Comment:
This feels wrong but I couldn't figure out how else to do it.

I'm not sure if this is good practice at all...